packages/frint-router/src/matchFromHistory.js

```diff
@@ -0,0 +1,40 @@
+import _ from 'lodash';
+import pathToRegexp from 'path-to-regexp';
+
+export default function matchFromHistory(pattern, history, options = {}) {
+  if (!pattern) {
+    return {
+      url: history.location.pathname,
```

Comment:
I think `history.location` will always exist. At least it was always the case in unit tests with `MemoryRouterService` even before pushing any URLs in it. The idea here is to return any match if there's no pattern to match. @fahad19, am I correct?